iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -324,6 +324,106 @@ TEST_F(PoshRuntime_test, GetMiddlewareReceiverReceiverlistOverflow)
     EXPECT_TRUE(receiverlistOverflowDetected);
 }
 
+TEST_F(PoshRuntime_test, GetMiddlewarePublisherIsSuccessful)
+{
+    const auto publisherPort = m_runtime->getMiddlewarePublisher(
+        iox::capro::ServiceDescription(99U, 1U, 20U), 0U, m_runnableName, iox::runtime::PortConfigInfo(11U, 22U, 33U));
+
+    ASSERT_NE(nullptr, publisherPort);
+    EXPECT_EQ(iox::capro::ServiceDescription(99U, 1U, 20U), publisherPort->m_serviceDescription);
+    /// @todo #25 historyCapacity no member?
+    /// @todo #25 memoryInfo not used in publisher port right now
+    // EXPECT_EQ(22u, publisherPort->m_memoryInfo.deviceId);
+    // EXPECT_EQ(33u, publisherPort->m_memoryInfo.memoryType);
```

Comment:
Please don't check in dead code ;-) I don't understand what's the issue here?

Reply:
See above regarding `@todo #25` tags.